lang/cs/Org.Apache.REEF.Examples/MachineLearning/KMeans/KMeansDriverHandlers.cs

```diff
@@ -78,7 +79,7 @@ public KMeansDriverHandlers()
                 _executionDirectory); 
 
             _totalEvaluators = _partitionsNumber + 1;
-            _mpiDriver = new MpiDriver(Identifier, Constants.MasterTaskId, new AvroConfigurationSerializer());
+            _mpiDriver = new MpiDriver(Identifier, Constants.MasterTaskId, _fanOut, new AvroConfigurationSerializer());
```

Comment:
It's not obvious from the current examples and test cases but maybe we should change them to inject the MpiDriver into the start handler.

We do this on the Task side when we inject the MpiClient into the task, so it would make sense to do it on the driver as well.